site/es/blog/new-in-devtools-108/index.md

```diff
@@ -69,36 +80,46 @@ function bar() {
 ```
 
 <!-- Transpilers and minifiers may turn them into comma-separated expressions. -->
+Transpiladores y minificadores pueden convertir el código anterior en expresiones separadas por comas.
 
 ```js
 function bar(){return foo(),foo(),42}
-``` 
+```
 
 <!-- This creates confusion during debugging because the stepping behavior is different between minified and authored code. It is even more confusing when using sourcemaps to debug the minified code in terms of the original code, as the developer is then looking at semicolons (which were under the hood turned into commas by the toolchain) but the debugger doesn't stop on them. -->
+Esto crea confusión durante la depuración porque el comportamiento paso a paso es diferente entre el código minificado y el creado. Es aún más confuso cuando se utilizan mapas de origen para depurar el código minimizado en términos del código original, ya que el desarrollador está mirando los puntos y comas (que bajo el capó se convirtieron en comas por la cadena de herramientas), pero el depurador no se detiene en ellos. .
```

Comment:
A suggestion on not translating the "under the hood" expression nor "sourcemaps" + extra period :D
```suggestion
Esto crea confusión durante la depuración porque el comportamiento paso a paso es diferente entre el código minificado y el creado. Es aún más confuso cuando se utilizan sourcemaps para depurar el código minimizado en términos del código original, ya que el desarrollador está mirando los puntos y comas (que se convirtieron en comas por la cadena de herramientas), pero el depurador no se detiene en ellos.
```